docs/guides/develop-a-module-in-isolation.md

```diff
@@ -167,52 +171,79 @@ export function App() {
+The `dev-local` script is similar to the `dev` script but introduces a `LOCAL` environment variable. This new environment variable will be utilized by the `webpack.dev.js` file to conditionally setup the development server for local development in isolation or to be consumed by a host application through the `/remoteEntry.js` entry point:
 
-```js !#3,7,11-13 remote-module/webpack.config.js
-import { remoteTransformer } from "@squide/webpack-module-federation/configTransformer.js";
+```js !#9,12 remote-module/webpack.dev.js
+// @ts-check
 
-const isLocal = env.LOCAL === "true";
+import { defineDevRemoteModuleConfig } from "@squide/webpack-module-federation/defineConfig.js";
+import { defineDevConfig } from "@workleap/webpack-configs";
+import { swcConfig } from "./swc.dev.js";
 
-/** @type {import("webpack").Configuration} */
-let config = {
-    entry: isLocal ? "./src/index.tsx" : "./src/register.tsx",
-    ...
-};
+let config;
 
-if (!isLocal) {
-    config = remoteTransformer(config, "remote1");
+if (!process.env.LOCAL) {
+    config = defineDevRemoteModuleConfig(swcConfig, "remote1", 8081);
+} else {
+    config = defineDevConfig(swcConfig);
 }
 
 export default config;
 ```
 
 Start the local application by running the `dev-local` script. The federated application shell should wrap the content of the index route of the module.
 
+[!ref icon="mark-github" text="View a remote module project example on Github"](https://github.com/gsoft-inc/wl-squide/tree/main/sample/remote-module)
+
 ## Local module
 
 Similarly to remote modules, you can achieve the same isolated setup for local modules. The main difference is that the `webpack.config.js` file of a local module serves the sole purpose of starting a development server for isolated development. Typically, local modules do not rely on [Module Federation](https://webpack.js.org/concepts/module-federation/).
 
+First open a terminal at the root of local module project and install the `@workleap/webpack-configs` package and it's dependencies:
+
++++ pnpm
+```bash
+pnpm add -D @workleap/webpack-configs @workleap/swc-configs webpack webpack-dev-server webpack-cli swc/core @swc/helpers browserslist postcss
+```
++++ yarn
+```bash
+yarn add -D @workleap/webpack-configs @workleap/swc-configs webpack webpack-dev-server webpack-cli swc/core @swc/helpers browserslist postcss
+```
++++ npm
+```bash
+npm install -D @workleap/webpack-configs @workleap/swc-configs webpack webpack-dev-server webpack-cli swc/core @swc/helpers browserslist postcss
+```
++++
+
+Then, perform the same steps as for [setuping an isolated setup for a remote module](#remote-module), except for the `dev-local` script and the [webpack](https://webpack.js.org/) configuration as they will diverge.
+
+Once you performed the remote module steps, add a new `dev-local` script to the `package.json` file to start the local development server:
+
 ```json local-module/package.json
 {
     "dev-local": "webpack serve --config webpack.config.js"
```

Comment:
it should not be `webpack.dev.js` here too?

Reply:
We could, I hesitated. I used `webpack.config.js` instead of `webpack.dev.js` because there will never be multiple webpack configurations for a local module, it will always be no webpack configuration or only a single dev configuration so I defered to the standard name for a webpack config which is `webpack.config.js`. Should I go for `webpack.dev.js` ?

Reply:
Ha, ok, it convinced me 👍